projects/laji/src/app/shared-modules/select-collections/pipe/collection-url.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { IdService } from '../../../shared/service/id.service';
+
+@Pipe({
+  name: 'collectionUrl'
+})
+export class CollectionUrlPipe implements PipeTransform {
+
+  transform(value: any): any {
+    return IdService.getUri(value);
+  }
+
+}
```

Comment:
This is almost exactly the same as `toFullUri` pipe. You can remove this and use that instead in all places where you're using `collectionUrl` (output is exactly the same).

Reply:
Ok, will do.

Reply:
Fixed now.